images/generate-image-certs.sh

```diff
@@ -9,7 +9,7 @@ openssl req -new -newkey rsa:2048 -nodes -keyout server.key -out server.csr \
     -subj '/O=Cockpit/OU=Cockpituous/CN=cockpit-tests' \
     -extensions server_ca_extensions -reqexts server_ca_extensions \
     -config "$OPENSSL_CNF"
-openssl x509 -req -days 365000 -in server.csr -out server.pem \
+openssl x509 -req -days 720 -in server.csr -out server.pem \
```

Comment:
Everything I know says that this maxes out at a year and a month or so, ~395 days.

https://support.globalsign.com/ssl/general-ssl/397-day-maximum-tls-certificate-validity

Not like it matters or anything, though.  We use this for curl, and in case we use this in a browser, we manually accept it anyway.  I'd say either make it a year ("correctness" for some value of correct), or leave it at 100 ("convenience" for some value of convenient).

Reply:
Good point. This is totally unimportant, I dropped the commit.